src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -419,99 +337,114 @@ def getCEStatus(self):
 
+        # Get a proxy
         result = self._prepareProxy()
         if not result["OK"]:
-            self.log.error("AREXComputingElement: failed to set up proxy", result["Message"])
+            self.log.error("Failed to set up proxy", result["Message"])
             return result
-        self.s.cert = Locations.getProxyLocation()
+        self.session.cert = Locations.getProxyLocation()
 
-        # Try to find out which VO we are running for. Essential now for REST interface.
+        # Try to find out which VO we are running for.
+        # Essential now for REST interface.
         res = getVOfromProxyGroup()
         vo = res["Value"] if res["OK"] else ""
 
-        result = S_OK()
-        result["SubmittedJobs"] = 0
+        # Prepare the command
         command = "info"
         params = {"schema": "glue2"}
         query = self.base_url + command
-        r = self.s.get(query, headers=self.headers, params=params, timeout=5.0 * self.arcRESTTimeout)
 
-        if not r.ok:
+        # Submit the GET request
+        response = self.session.get(query, headers=self.headers, params=params, timeout=self.arcRESTTimeout)
+        if not response.ok:
             res = S_ERROR("Unknown failure for CE %s. Is the CE down?" % self.ceHost)
             return res
-
-        p = json.loads(r.text)
+        ceData = response.json()
 
         # Look only in the relevant section out of the headache
-        info = p["Domains"]["AdminDomain"]["Services"]["ComputingService"]["ComputingShare"]
+        queueInfo = ceData["Domains"]["AdminDomain"]["Services"]["ComputingService"]["ComputingShare"]
+        if not isinstance(queueInfo, list):
+            queueInfo = [queueInfo]
 
         # I have only seen the VO published in lower case ...
+        result = S_OK()
+        result["SubmittedJobs"] = 0
+
         magic = self.queue + "_" + vo.lower()
-        for i in range(len(info)):
-            if info[i]["ID"].endswith(magic):
-                result["RunningJobs"] = info[i]["RunningJobs"]
-                result["WaitingJobs"] = info[i]["WaitingJobs"]
+        for i in range(len(queueInfo)):
+            if queueInfo[i]["ID"].endswith(magic):
+                result["RunningJobs"] = queueInfo[i]["RunningJobs"]
+                result["WaitingJobs"] = queueInfo[i]["WaitingJobs"]
                 break  # Pick the first (should be only ...) matching queue + VO
 
         return result
 
     #############################################################################
+
     def _renewJobs(self, jobList):
         """Written for the REST interface - jobList is already in the REST format
-        This function is called only by this class, NOT by the SiteDirector"""
-        for job in jobList:
+        This function is called only by this class, NOT by the SiteDirector
+        """
+
+        # List of jobs in json format for the REST query
+        jList = [self._DiracToArcID(job) for job in jobList]
```

Comment:
Is this line needed given that the jobList is already in the format we need?

Reply:
No you are right, I didn't see that the method was protected and not usable outside of a specific context (`getJobStatus()`).
I am going to change that, thanks.